test/testHeadless.m

```diff
@@ -18,9 +18,12 @@
 % The width and height are specified to circumvent different DPIs in developer
 % machines. The float format reduces the probability that numerical differences
 % in the order of numerical precision disrupt the output.
-    extraOptions = {'width' ,'\figurewidth' ,...
-                    'height','\figureheight',...
-                    'floatFormat', '%8.6g'  ,...
+    extraOptions = {'width' ,'\figureWidth', ...
+                    'height','\figureHeight',...
+                    'floatFormat', '%8.3f',  ...
```

Comment:
I changed this on purpose to avoid number close to zero resulting in something like `1.233232e-26`. Those are now `0.000`. Or is do you have a reason for sticking with `%g`?